articles/libraries/_includes/_configure_embedded_login.md

```diff
@@ -0,0 +1,10 @@
+
+### Configure Auth0 for Embedded Login
+
+Lock v11 and Auth0.js v9 use cross-origin calls inside hidden iframes to perform authentication. To make sure this can be done securely, the Auth0 sever needs to know the domains where you will be hosting your applications.
+
+Add the domain to the **Allowed Web Origins** field. You can find this field in the [Client Settings](${manage_url}/#/clients/${account.clientId}/settings).
```

Comment:
I would be a little more specific then just _domain_ as that setting requires the user to set a web origin as defined by the spec (scheme, host and port when using a non-default port for the scheme). Ideally, this should have two examples, one with default ports and another with non-default ports. Something like:

* if you're running your application at `https://example.com/portal` then you need to add the web origin `https://example.com`
* if you're running your application at `https://example.com:5503/portal` then you need to add the web origin `https://example.com:5503`

An explicit warning that the path component should not be included even if it's just `/` might be useful as I already seen people adding `https://example.com/` when it should not contain the last `/`.